docs/develop/node/running-a-node.md

```diff
@@ -188,3 +188,54 @@ To check logs call: `nearup logs` or `nearup logs --follow`
 >Got a question?
 <a href="https://stackoverflow.com/questions/tagged/nearprotocol">
```

Comment:
The "Got a question" prompt with link to SO should be at the bottom of the document.